content/en/blog/2020/custom-istio-dns-name/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Customizing Istio DNS stub names"
+subtitle: Changing or adding stub domains to make ServiceEntry hostnames routable
+description: Customizing global or region hostnames as declared in ServiceEntry(s) for routability.
+publishdate: 2020-05-05
+attribution: "Christian Posta (Solo.io)"
+keywords: [istio, dns, mTLS, coredns]
+---
+
+Istio can be run across multiple clusters and has the necessary configuration APIs to support [shared control plane](/docs/setup/install/multicluster/shared/) or a non-shared, [replicated control model](/docs/setup/install/multicluster/gateways). The motivation for deploying across multiple clusters has been covered excellently by fellow Istio contributor [Vadim Eisenberg](https://twitter.com/vadimeisenberg) in the [Multi-Mesh Deployments for Isolation and Boundary Protection](/blog/2019/isolated-clusters/) blog.
+
+One aspect of multi-cluster functionality enabling services to communicate with each other is service discovery. In an Istio replicated control-plane multi-cluster setup, you create [ServiceEntry](/docs/reference/config/networking/service-entry/)s in one cluster to point to services running in another cluster. For example:
```

Comment:
control plane has no dash.

Reply:
`you create [Service Entry](link) custom resources in one cluster to reference services running in another cluster.`

Reply:
> control plane has no dash.

Control plane should have a hyphen when it describes "setup" a noun and is linked ...(https://www.grammarbook.com/punctuation/hyphens.asp) Unless this is some convention used purposefully throughout the docs?

Reply:
Istio uses this style guide, which I quite like: https://developers.google.com/style/hyphens. I think it well-represents the expectations of the technical writer community. My writing is not pro enough to be considered such a member, but I try :)

Reply:
> `you create [Service Entry](link) custom resources in one cluster to reference services running in another cluster.`

Agreed. In particular, it's best to avoid making a keyword into a plural; for more on that, see (https://developers.google.com/style/code-in-text#keywords).